Clover/app/src/main/java/org/floens/chan/core/model/PostImage.java

```diff
@@ -23,7 +23,7 @@
 
 public class PostImage {
     public enum Type {
-        STATIC, GIF, MOVIE
+        STATIC, GIF, MOVIE, SWF
```

Comment:
Maybe we can call it OTHER too since I see no future in playing swf files on Android.

Reply:
see PR # 780 
i could however change everything up if you believe it's a cleaner aproach.

Reply:
If there are apps for that maybe we can integrate it with those apps? But sure let's keep SWF then.